locales/de-DE.json

```diff
@@ -2175,8 +2175,8 @@
     },
     "elevator": {
       "phrases": {
-        "0": "Take the elevator.",
-        "1": "Take the elevator to <LEVEL>."
```

Comment:
Isn't this missing the string "level"? As in `Take the elevator to level <LEVEL>.`

Reply:
It looks like maybe it's hardcoded in https://github.com/valhalla/valhalla/blob/master/src/odin/enhancedtrippath.cc#L535, though that doesn't look to be localized. I'm not very familiar with this codebase though.

Reply:
Thx for the code hint, was too lazy to look it up so far, just had a harder look. You're right, the string "Level" is not localized. I guess that's because it's not necessarily referring to a numbered level, but could be just a string with the `level:ref=<string>` OSM tag, instead of the usual `level=<int>` tag. Guess that could be added to the localization and processing it, but not sure it's really worth the effort? Anyways, had a bit deeper glimpse into the narration code, which is good:)

Reply:
Yeah I think the general idea is that it could be any name not just a number or something so putting level there all the time might not always make sense. To handle both cases properly we would need to detect the presence of a number like name which we could do for sure at some point